asn1tools/codecs/per.py

```diff
@@ -1399,6 +1399,12 @@ def __init__(self, name, values, numeric):
         self.root_data_to_index = data_to_index
         self.root_number_of_bits = integer_as_number_of_bits(len(index_to_data) - 1)
 
+        if numeric:
+            self.root_data_to_value = {k: k for k in enum_values_as_dict(root)}
+        else:
+            self.root_data_to_value = \
+                {v: k for k, v in enum_values_as_dict(root).items()}
```

Comment:
Use parentesis instead of `\` to match the styling of the rest of the package. This allies throughout the PR.

``` python
            self.root_data_to_value = (
                {v: k for k, v in enum_values_as_dict(root).items()})
```

Reply:
Well, in this case, the line was exactly 90 characters, so it was not required. But I changed the other occurrences.